views/home.html

```diff
@@ -0,0 +1,26 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/public/style.css">
+    <title>Home</title>
+</head>
+<body>
+    <h1>Clark Kent</h1>
+    
+        <div>
+            <p>This page is all about the flying superhero - Superman!</p>
+        </div>
+
+        <img id="superman" src="/images/Superman-Year-One_Featured.jpg" alt="superman">
+
+        <div>
+            <ul>
+                <li><a href="/about"></a>About</li>
```

Comment:
For it to work the About should be inside the ancor tag like `<a href="/about">About</a>`